tests/test_project.py

```diff
@@ -1,4 +1,6 @@
-import pytest
+import os
```

Comment:
I think I would like to add a  more clear distinction between tests that work on the template and tests that work on a generated package. My suggestion would be to add to subdirectories under `tests/`, like so

1. `tests/generated-package/`
2. `tests/template/`

Reply:
I think this distinction is not that clear cut. All tests we have now use bake() to generate a project and do test against it. 
So making separate dirs will not help.

Reply:
Let's leave it for now